src/icons/__stories__/mistica-icons-story.tsx

```diff
@@ -19,22 +18,39 @@ const misticaIcons = ((requireContext) => {
         component.componentName = fileNameToComponentName(id);
         return component;
     });
-})(require.context('../../generated/mistica-icons/', true, /\.tsx$/));
+})(require.context('../../generated/mistica-icons/', true, /^(?!\.\/icons\-keywords\.tsx$).+\.(?:tsx)$/));
```

Comment:
Ignore the file containing the keywords (`icons-keywords.tsx`)

Reply:
you could have moved the file inside a subfolder to avoid this